### PR TITLE
Grouped dependencies

### PR DESCRIPTION
**Pull request by SecUpwN**
This PR groups our dependencies in `build.gradle` and adds a small description above each group.

### CODE REVIEW

File: AIMSICD/build.gradle [hunk 1]
```diff
@@ -83,23 +83,36 @@ android {
...
 
-    //This git hash resolves to version 5.5
-    compile 'com.github.MKergall.osmbonuspack:OSMBonusPack:2e8bca20f7'
+    // OSMBonusPack library to interact with OpenStreetMap data
+    compile 'com.github.MKergall.osmbonuspack:OSMBonusPack:v5.6'
+    
+    // Transition library for animated transition effects
     compile 'com.github.kaichunlin.transition:core:0.9.2'
-    compile 'io.freefair.android-util:logging:1.1.0'
-    //debugCompile 'com.squareup.leakcanary:leakcanary-android:1.3.1'
```
Comment (larsgrefer) [inline AIMSICD/build.gradle]: If you remove leakcanary here you should remove it in the Application class too

Comment (SecUpwN) [inline AIMSICD/build.gradle]: Thanks for reviewing, @larsgrefer. Do you agree that we can ditch leakcanary if we implement our new stracktrace collection solution? I just want to remove unnessary cruft from our app.

Comment (larsgrefer) [inline AIMSICD/build.gradle]: No, I don`t agree. LeakCanary and *our new stacktrace collection solution* are two different things, none of them makes the other one obsolete

File: AIMSICD/src/main/java/com/SecUpwN/AIMSICD/AppAIMSICD.java [hunk 2]
```diff
@@ -46,8 +46,6 @@ public void onCreate() {
         addModule(new AndroidLoggerModule());
         addModule(OkHttpModule.withCache(this));
         super.onCreate();
-        // DO NOT REMOVE BELOW COMMENTED-OUT CODE BEFORE ASKING!
```
Comment (larsgrefer) [inline AIMSICD/src/main/java/com/SecUpwN/AIMSICD/AppAIMSICD.java]: Don't forget lines 26-27 in this file :stuck_out_tongue_winking_eye:

File: AIMSICD/build.gradle (hunk 1 above)
Comment (SecUpwN) [inline AIMSICD/build.gradle]: >No, I don`t agree.

WTF, shall I re-add it?

Comment (larsgrefer) [inline AIMSICD/build.gradle]: I'm not not sure if we might need it, but let's keep it away for now.
Im not a fan of commented-out code.
If we want to use it in future, we can re-add it in future.

Comment (SecUpwN) [inline AIMSICD/build.gradle]: Thanks for your clear words. Fun to develop with you. :+1:

**Pull request merged**